packages/react-integration/README.md

```diff
@@ -53,10 +55,10 @@ Run the following command from the root directory to launch Cypress:
 
 You can leave both the demo app and Cypress running while you develop your tests. You may receive notifications from Cypress that ask you to refresh as you make changes. Use the “Try again” button to refresh Cypress with your updated tests.
 
-Add any new integration tests you write to the [integration folder here](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration/cypress/integration).
-
+Add any new integration tests you write to the [integration folder here](https://github.com/patternfly/patternfly-react/tree/master/packages/react-integration/cypress/integration).
```

Comment:
Is it possible to use a relative path here? I think they generally work better when navigating the directory structure in someone's fork of the repo, makes it easier to preview your changes before submitting the PR.

Reply:
Hmm, yeah we should be able to. I just did a find-and-replace, didn't notice some of them aren't relative.

Reply:
Pushed an update to use relative paths here and in a few other places!